docs/dymos_book/features/phases/analytic_phases.ipynb

```diff
@@ -0,0 +1,448 @@
+{
```

Comment:
<p>I wonder if it is too limiting to prohibit time-varying control.  Just from the point of view of thinking of this as an openmdao black box that is pretending to be a phase. I don't have an example or usecase though.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenMDAO/dymos/pull/833/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
I've been thinking about this and I'm starting to be able to see some cases where controls might work.
Say you had some input at each node and wanted to integrate it with a trapezoidal rule.
However, since the nodes are only at the LGL nodes *in phase space* (not in segment space) this would only allow for polynomial controls.

Reply:
On second thought I don't like this quite so much, because then the density of the nodes affects the solution, which is NOT the use-case for Analytic phases.